src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -2269,4 +2269,39 @@ public Object executeScript(String script) {
         return getSeleniumHelper().executeJavascript(statement);
     }
 
+    /**
+     * @param place to check
+     * @return true if content of place is valid (according to validation pattern)
+     */
+    public boolean valueOfIsValid(String place) {
+        T element = getElementToRetrieveValue(place, null);
+        List<T> invalidElements = findAllByCss("input:invalid");
+        Collection<String> attributesToCompare = Arrays.asList("name", "id");
```

Comment:
Does this work for elements that don't have an id or name?

Reply:
I expect not.